src/sentry/api/endpoints/organization_projects.py

```diff
@@ -82,11 +82,11 @@ def get(self, request: Request, organization) -> Response:
         """
         Return a list of projects bound to a organization.
         """
-        stats_period = request.GET.get("statsPeriod")
+        stats_period = request.GET.get("statsPeriod", request.GET.get("timeframe"))
```

Comment:
If a client provides both timeframe and statsPeriod should we prefer the new key instead of the old one?

Reply:
good point. I was biased towards minimizing any potential of a breaking change but it should be the other way around for sure. will fix